src/config.rs

```diff
@@ -73,6 +73,9 @@ pub struct Config {
     /// accessing `ctx.config.settings`.
     #[serde(default)]
     pub settings: Option<serde_json::Value>,
+
+    #[serde(skip)]
+    pub path: Option<PathBuf>,
```

Comment:
I think we can remove it, and print only the environment name

Reply:
Thank you! I have removed the path field and updated the code to print only the environment name.